internal/spinner/spinner.go

```diff
@@ -25,9 +25,13 @@ var (
 	Duration = 100 * time.Millisecond
 )
 
+type Spinner struct {
```

Comment:
Why wrapping it in our own type?

Reply:
otherwise i will have to import the backing library to be able to do [this](https://github.com/finleap-connect/monoctl/pull/116/files#diff-e5c51dd2e4d6f31d8605824672fe197760488fb1266dddb65eace16e6b1d0413R81) which will result in "2" spinner improts, which I didn't like. By wrapping it we keep the backing library isolated.